container/build.sh

```diff
@@ -27,8 +26,8 @@ while [ -n "$1" ]; do
         usage >&2
         exit 0
         ;;
-    -t|--tag)
-        TAG="$2"
+    -es|--es-version)
```

Comment:
I was debating if there would be any value in keeping `tag` as an additional option but I don't think so i.e. even if developers want to play around and make changes they can just use `--es-version` to produce an image with a different tag and that seems appropriate.